plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -12,7 +13,7 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
     super(...arguments);
 
     this.state = {
-      filteredNodes: [],
+      filteredNodes: new NodesList([]),
```

Comment:
Where is this state used?

Reply:
This state was removed but I readded this.